docs/docs/articles/exposing-testkube-with-ingress-nginx.md

```diff
@@ -2,85 +2,92 @@
 
 ## Prerequisites
 
-Add the repo to Helm:
+1. Deploy NGINX Ingress Controller into your k8s cluster. Please see the link for more details: [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/).
 
-```sh
-helm repo add kubeshop https://kubeshop.github.io/helm-charts && helm repo update
-```
-
-A values file for guidance can be found [here](https://github.com/kubeshop/helm-charts/blob/39f73098630b333ba66db137e7fc016c39d92876/testkube/charts/testkube/values-demo.yaml).
-
-## Configure Ingress-NGINX to Expose Testkube API
-
-The Testkube Dashboard needs the Testkube API to be exposed. For this, you can run:
-
-```sh
-helm upgrade testkube kubeshop/testkube --set testkube-api.ingress.enabled="true"
-```
-
-By default, the API's entry point is the path `/results`, so the results will be accessible at `$INGRESS_HOST/results/`
-
-The Ingress configuration used is available in the [Testkube Helm Repo](https://github.com/kubeshop/helm-charts).
-
-## Exposing the Testkube Dashboard 
-
-To expose the Dashboard and the API together, run: 
-
-```sh
-helm install testkube kubeshop/testkube --set testkube-dashboard.enabled="true" --set testkube-dashboard.ingress.enabled="true" --set testkube-api.ingress.enabled="true"
-```
-
-To get the address of Ingress use:
+2. (Optional) To use TLS it is required to have any certificate management tool installed. At Testkube and in this guide we will use [cert-manager](https://cert-manager.io/), but it might differ depending on your set-up.
```

Comment:
```suggestion
2. (Optional) To use TLS, the installation of any certificate management tool is required. At Testkube and in this guide, we will use [cert-manager](https://cert-manager.io/), but it might differ depending on your set-up.
```